src/features/projectsV2/ProjectDetails/components/microComponents/SampleTrees.tsx

```diff
@@ -1,23 +1,38 @@
+import { useTranslations } from 'next-intl';
+import { SetState } from '../../../../common/types/common';
 import { SamplePlantLocation } from '../../../../common/types/plantLocation';
 import styles from '../../styles/PlantLocationInfo.module.scss';
-import { useTranslations } from 'next-intl';
 
 interface Props {
   sampleInterventions: SamplePlantLocation[];
+  setSelectedSamplePlantLocation: SetState<SamplePlantLocation | null>;
 }
-const SampleTreeList = ({ sampleInterventions }: Props) => {
+const SampleTreeList = ({
+  sampleInterventions,
+  setSelectedSamplePlantLocation,
+}: Props) => {
   const tProjectDetails = useTranslations('ProjectDetails');
+  const selectSampleSpecies = (speciesName: string) => {
+    if (!speciesName) return;
+    const result = sampleInterventions.find(
+      (sampleTree) => speciesName === sampleTree.scientificName
+    );
+    if (result) setSelectedSamplePlantLocation(result);
+  };
```

Comment:
I don't get why we're trying to select a sample species - are we not selecting a samplePlantLocation?

Reply:
We should call it 'samplePlant' instead of 'samplePlantLocation' because we are displaying plant information (i.e., measurements, scientific name, etc.) rather than location information